fulp_modules/features/antagonists/bloodsuckers/code/powers/_powers.dm

```diff
@@ -139,12 +139,15 @@
 				return FALSE
 	return TRUE
 
-/// NOTE: With this formula, you'll hit half cooldown at level 8 for that power.
 /datum/action/cooldown/bloodsucker/StartCooldown()
-	// Calculate Cooldown (by power's level)
-	if(power_flags & BP_AM_STATIC_COOLDOWN)
+	// Check for power flags first
+	if(!power_flags)
+		return
+	if(BP_AM_STATIC_COOLDOWN)
```

Comment:
You removed "power_flags &" here?

Reply:
I might've mistaken `&` for `&&`, I'll fix that. What exactly does `&` do in this context by the way? Admittedly my overall coding knowledge is pretty amateurish, and I'm only familiar with most of the basic operators...

Reply:
& is to check if a flag is on, && is "and" between 2 different things to check

Reply:
Okay, thank you very much! :D